docs/docs/1.3.0/quickstart/universal.md

```diff
@@ -166,19 +219,9 @@ metrics:
 EOF
 ```
 
-This will enable the `prometheus` metrics backend on the `default` [Mesh](/docs/1.3.0/policies/mesh/) and automatically collect metrics for all of our traffic.
-
-Now let's go ahead and generate some traffic - to populate our charts - by using the demo application!
+This will enable the `prometheus` metrics backend on the `default` [Mesh](/docs/1.2.3/policies/mesh/) and automatically collect metrics for all of our traffic.
 
-:::tip
-You can also generate some artificial traffic with the following command to save some clicks:
-
-```sh
-while [ true ]; do curl http://192.168.33.70:8000/items?q=; curl http://192.168.33.70:8000/items/1/reviews; done
-```
-:::
-
-And then access the Grafana dashboard at [192.168.33.80:3000](http://192.168.33.80:3000/) with default credentials for both the username (`admin`) and the password (`admin`).
+Increment the counter to generate traffic, and access the dashboard at [127.0.0.1:3000](http://127.0.0.1:3000) with default credentials for both the username (`admin`) and the password (`admin`).
```

Comment:
The dashboard can be exposed on whathever port user started grafana.

Should we also drop a sentence that a user needs to configure Prometheus / Grafana using TrafficMetrics docs?